01/app.js

```diff
@@ -0,0 +1,26 @@
+const formEl = document.querySelector('form');
+const ulEl = document.querySelector('ul');
+const firstNameEl = formEl.querySelector('input[name="firstName"]');
+const lastNameEl = formEl.querySelector('input[name="lastName"]');
+
+formEl.addEventListener('submit', handleInput);
+
+function handleInput(e) {
+    e.preventDefault();
+
+    let firstName = firstNameEl.value;
+    let lastName = lastNameEl.value;
+
+    printFullName(firstName, lastName);
```

Comment:
Skoro tu jest funkcja to i poniżej można by było stworzyć i wykorzystać np. `clearInputValue()` ;)